src/settingsmanager.h

```diff
@@ -22,22 +22,18 @@ class SettingsManager : public QObject
     bool settingsExists(const QString &key);
     QVariant getSettings(const QString &key);
     qreal getZoomFactorByZimId(const QString &id);
-    int getKiwixServerPort() const { return m_kiwixServerPort; };
-    QString getKiwixServerIpAddress() const { return m_kiwixServerIpAddress; };
-    qreal getZoomFactor() const { return m_zoomFactor; };
-    QString getDownloadDir() const { return m_downloadDir; };
-    QString getMonitorDir() const { return m_monitorDir; };
-    QStringList getMonitorDirZims() const { return m_monitorDirZims; };
+    int getKiwixServerPort() const { return m_kiwixServerPort; }
+    QString getKiwixServerIpAddress() const { return m_kiwixServerIpAddress; }
+    qreal getZoomFactor() const { return m_zoomFactor; }
+    QString getDownloadDir() const { return m_downloadDir; }
+    QString getMonitorDir() const { return m_monitorDir; }
+    QStringList getMonitorDirZims() const { return m_monitorDirZims; }
 
 public slots:
     void setKiwixServerPort(int port);
-    int getKiwixServerPort() { return m_kiwixServerPort; };
     void setKiwixServerIpAddress(QString ipAddress);
-    QString getKiwixServerIpAddress() { return m_kiwixServerIpAddress; };
     void setZoomFactor(qreal zoomFactor);
-    qreal getZoomFactor() { return m_zoomFactor; };
     bool setDownloadDir(QString downloadDir);
-    QString getDownloadDir() { return m_downloadDir; }
     bool setMonitorDir(QString monitorDir);
```

Comment:
This commit ("Move getters in settingsmanager out of slots") looks strange. Consider `getKiwixServerPort()`. The commit says that there were two (const non-slot and non-const slot) versions of it. The non-const variant was dropped, and for the const variant there was only a small punctuation fix.

Reply:
I must've dropped the non-const variant by mistake. But I don't really know what you suggest to do here (except bringing the non const variant back?). Any hints?

Reply:
> I must've dropped the non-const variant by mistake.

Non-const slot variants were converted to const non-slots based on [my suggestion](https://github.com/kiwix/kiwix-desktop/pull/743#discussion_r788023785). Then you also performed some [cosmetic clean-up](https://github.com/kiwix/kiwix-desktop/pull/743#discussion_r792698362). During history cleanup those two changes were combined in a way that doesn't accurately reflect what has happened.